src/networks.json

```diff
@@ -1,673 +1,683 @@
-    "chainId": 1285,
-    "network": "mainnet",
-    "multicall": "0x537004440ffFE1D4AE9F009031Fc2b0385FCA9F1",
-    "rpc": ["https://rpc.moonriver.moonbeam.network"],
-    "explorer": "https://blockscout.moonriver.moonbeam.network/"
-  },
-    "25": {
-    "key": "25",
-    "name": "Cronos Mainnet",
-    "shortName": "Cronos",
-    "chainId": 25,
-    "network": "mainnet",
-    "multicall": "0x6F522a3982e8F9A50A2EDc9E46ed1A3aE2B3FD3a",
-    "rpc": ["https://evm-cronos.crypto.org"],
-    "explorer": "https://cronos.crypto.org/explorer"
-  },
-    "44": {
-    "key": "44",
-    "name": "Crab Network",
-    "shortName": "Crab",
-    "chainId": 44,
-    "network": "mainnet",
-    "multicall": "0x4617D470F847Ce166019d19a7944049ebB017400",
-    "rpc": ["https://crab-rpc.darwinia.network"],
-    "explorer": "https://crab.subscan.io/"
-  },
-  "288": {
-    "key": "288",
-    "name": "Boba Mainnet",
-    "shortName": "Boba",
-    "chainId": 288,
-    "network": "mainnet",
-    "multicall": "0xfb91c019D9F12A0f9c23B4762fa64A34F8daDb4A",
-    "rpc": ["https://mainnet.boba.network/"],
-    "explorer": "https://blockexplorer.boba.network/"
-  },
-  "5553": {
-    "key": "5553",
-    "name": "Nahmii Testnet",
-    "shortName": "Nahmii",
-    "chainId": 5553,
-    "network": "testnet",
-    "testnet": true,
-    "multicall": "0x0e157d2E45af27564edFAaCcD68f2f0458F3D96c",
-    "rpc": ["https://l2.testnet.nahmii.io"],
-    "explorer": "https://explorer.testnet.nahmii.io/"
-  },
-    "5551": {
-    "key": "5551",
-    "name": "Nahmii Mainnet",
-    "shortName": "Nahmii",
-    "chainId": 5551,
-    "network": "mainnet",
-    "multicall": "0x05911151467b9F42eD14f10ddE0c057347Fff714",
-    "rpc": ["https://l2.nahmii.io"],
-    "explorer": "https://explorer.nahmii.io/"
-  },
-    "1088": {
-    "key": "1088",
-    "name": "Metis Andromeda Mainnet",
-    "shortName": "andromeda-mainnet",
-    "chainId": 1088,
-    "network": "mainnet",
-    "multicall": "0xc39aBB6c4451089dE48Cffb013c39d3110530e5C",
-    "rpc": ["https://andromeda.metis.io/?owner=1088"],
-    "explorer": "https://andromeda-explorer.metis.io/"
-  },
-  "361": {
-    "key": "361",
-    "name": "Theta Mainnet",
-    "shortName": "Theta",
-    "chainId": 361,
-    "network": "mainnet",
-    "multicall": "0xB48BbAD564Ceb6fB30cCea2Af248ccF6398aEab5",
-    "rpc": ["https://eth-rpc-api.thetatoken.org/rpc"],
-    "explorer": "https://explorer.thetatoken.org/"
-  },
-  "10000": {
-    "key": "10000",
-    "name": "smartBCH",
-    "shortName": "BCH",
-    "chainId": 10000,
-    "network": "mainnet",
-    "multicall": "0x1b38EBAd553f218e2962Cb1C0539Abb5d6A37774",
-    "rpc": ["https://smartbch.greyh.at/"],
-    "explorer": "https://www.smartscan.cash"
+    "1": {
+      "key": "1",
+      "name": "Ethereum Mainnet",
+      "chainId": 1,
+      "network": "homestead",
+      "multicall": "0xeefba1e63905ef1d7acba5a8513c70307c1ce441",
+      "ensResolver": "0x4976fb03C32e5B8cfe2b6cCB31c09Ba78EBaBa41",
+      "rpc": [
```

Comment:
Hello @mar1 Can you make the PR just have changes related to moonbeam network, it should not change entire file :)